### PR TITLE
Add notebooks submodule redo docker

### PR DESCRIPTION
**Pull request by alexgleith**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -8,14 +8,14 @@ on:
     paths:
       - "**"
 
-  release:
-    types:
-      - created
+  pull_request:
+    paths:
+      - "Dockerfile"
+      - "Requirements.txt"
```
Comment (dominicschaff) [inline .github/workflows/docker.yml]: Probably should be lowercase

File: cube-in-a-box-cloudformation.yml [hunk 2]
```diff
@@ -77,12 +77,15 @@ Resources:
...
-          make -C /home/ubuntu/cube-in-a-box-main setup-prod BBOX="${ExtentToIndex}"
+          apt-get update && apt-get install -y wget unzip git
+          cd /home/ubuntu
+          git clone https://github.com/digitalearthafrica/cube-in-a-box.git
+          cd cube-in-a-box
+          git submodule init
+          git submodule update
+          bash /home/ubuntu/cube-in-a-box/setup.sh
+          echo 'CIABPASSWORD=${SecretPassword}' > /home/ubuntu/cube-in-a-box/.env
+          make -C /home/ubuntu/cube-in-a-box setup-prod BBOX="${ExtentToIndex}"
```
Comment (dominicschaff) [inline cube-in-a-box-cloudformation.yml:88]: `setup-prod` doesn't use the local Dockerfiles. so are you sure we need git?

File: .github/workflows/docker.yml (hunk 1 above)
Comment (alexgleith) [reply .github/workflows/docker.yml]: Nice pickup

File: cube-in-a-box-cloudformation.yml (hunk 2 above)
Comment (alexgleith) [reply cube-in-a-box-cloudformation.yml:88]: Yeah we do, because there's a submodule for the notebooks now.

Comment (alexgleith) [reply cube-in-a-box-cloudformation.yml:88]: And it mounts the `./notebooks` folder as a volume

**Review by dominicschaff — CHANGES_REQUESTED** [review]
The workflow case must just be changed, then this PR is good.

**Comment by alexgleith** [comment]
> The workflow case must just be changed, then this PR is good.

Ah, sorry, I forgot to push the commit!

**Review by dominicschaff — APPROVED** [review]

**Pull request merged**